### PR TITLE
[WIP] Add blog article about storage honor reclaim policy

### PR DESCRIPTION
**Pull request by xing-yang**
Place holder blog PR for https://github.com/kubernetes/enhancements/issues/2644

### CODE REVIEW

**Comment by tengqm** [comment]
Instead of (in addition to) posting a blog, we may want to write some documentation on this feature.

xref: #32338

**Comment by mehabhalodiya** [comment]
> xref: https://github.com/kubernetes/website/pull/32338
 
@tengqm As there's already opened up PR by you, I think we should close this! Let me know your thoughts 🙂

**Comment by tengqm** [comment]
@mehabhalodiya I was expecting "more" from this PR. The PR I committed only has limited information about the feature. Users may want to know more about it, such as the design motivation, how it works, how to use it, any caveats they need to know, so on and so forth. These warrant some docs to be created.

I'm not against a blog post. Considering that a blog post may decay soon because we the doc team don't have enough resource to keep them updated, I was suggesting some more "formal" documentation about this feature. It can be a few paragraphs, with and without an example, or a new page somewhere.

**Comment by sftim** [comment]
I'll mark this as WIP
/retitle [WIP] Add blog article about storage honor reclaim policy

File: content/en/blog/_posts/2022-04-19-honor-reclaim-policy.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: blog
+title: "Kubernetes 1.24 Prevent PersistentVolume leaks when deleting out of order"
```
Comment (sftim) [inline content/en/blog/_posts/2022-04-19-honor-reclaim-policy.md:3]: An early nit
```suggestion
title: "Kubernetes 1.24: Prevent PersistentVolume leaks when deleting out of order"
```

**Comment by xing-yang** [comment]
@deepakkinni already opened a placeholder PR for blog here: https://github.com/kubernetes/website/pull/32453

I'm closing this one.

**Pull request closed without merging**